Extensions/one_click_postage.js

```diff
@@ -1485,25 +1426,62 @@ XKit.extensions.one_click_postage = new Object({
 		XKit.storage.set("one_click_postage", "already_reblogged", JSON.stringify(XKit.extensions.one_click_postage.already_reblogged));
 	},
 
-	show_error: function(code, state) {
-		var m_word = "reblog";
-		if (state === 1) { m_word = "draft"; }
-		if (state === 2) { m_word = "queue"; }
+	show_error: function(error, state) {
+		const verb = ["reblog", "draft", "queue"][state];
+		let message = "";
+
+		if (error.status) {
+			// HTTP error: response object
```

Comment:
no comments! if it's not clear enough, give it a utility function. (I think it's clear enough without the comments)